modules/sdk-core/src/bitgo/wallet/wallets.ts

```diff
@@ -605,6 +617,20 @@ export class Wallets implements IWallets {
     }
   }
 
+  /**
+   * Determine the multisig type based on the coin's capabilities.
+   * @param baseCoin
+   * @returns {string | undefined}
+   */
+  private getDefaultMultisigType(baseCoin: IBaseCoin): MultisigType | undefined {
+    if (baseCoin.supportsMultisig()) {
+      return multisigTypes.onchain;
+    } else if (baseCoin.supportsTss()) {
+      return multisigTypes.tss;
+    }
+    return undefined;
```

Comment:
this method should be in baseCoin cause for coins which supports both multisig and tss 
this method always returns `onchain` 
there is no option to default to `tss`